exercises/concept/fibonacci-iterator/.docs/introduction.md

```diff
@@ -85,7 +85,11 @@ Base.iterate(S::Squares) = iterate(S, 1)
 
 You may find it useful to combine these two definitions into one using optional arguments:
 
-## optional-arguments
+## Making use of optional arguments
```

Comment:
I feel like it makes sense to quickly go over optional arguments in the introduction for `fibonacci-iterator` again, even if it's technically already introduced in the new `high-score` exercise.

By the time the student gets to this, they may have forgotten and it's commonly used for defining iterators.